slacker/environment/config.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, quiet_mode=False):
       try:
         self.load()
       except Exception as ex:
-        self.__logger.debug('Config does not exist: {}\n{}'.format(self.__file_path(), ex))
+        self.__logger.debug('Config does not exist: {}\n{}'.format(self.file_path(), ex))
```

Comment:
Might be useful to bump this to INFO

Reply:
It's mostly for the first time running slacker and it hasn't got a config file yet.